common/test/acceptance/tests/lms/test_ccx.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""
+End-to-end tests for the CCX dashboard.
+"""
+from nose.plugins.attrib import attr
+
+from ...fixtures.course import CourseFixture
+from ..helpers import UniqueCourseTest, EventsTestMixin
+from ...pages.lms.auto_auth import AutoAuthPage
+from ...pages.lms.ccx_dashboard_page import CoachDashboardPage
```

Comment:
I think the last time I submitted a PR, they made me use absolute imports, despite what all the other files in this directory look like.